source/process/release-process.md

```diff
@@ -268,23 +284,33 @@ If a bug fix release is required, run through the following steps:
 
 Once final dot release build is ready to cut:
 
-1. Build:  
+1. Dev:
+    - Verify hashes and GPG signatures are correct, once build is cut
+    - Delete RCs after final version is shipped
+    - Merge changes made to release branch into vXXX
+2. Build:  
     - Tag a new release (e.g. 1.1.1) and run an official build  
     - Update CI servers, pre-release and GitLab Mattermost to the final version    
-2. PM:
+3. PM:
     - Update [Mattermost pricing page](https://about.mattermost.com/pricing/) if anything has changed
     - Merge the Changelog PR with notes on patch releases (see [example entry](https://docs.mattermost.com/administration/changelog.html#release-v3-5.1))
     - Submit GitLab MR to update the version number and MD5 hash to the dot release version. [See example](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/1127)
-      - [Test the upgrade](https://docs.google.com/document/d/1mbeu2XXwCpbz3qz7y_6yDIYBToyY2nW0NFZq9Gdei1E/edit#heading=h.ncq9ltn04isg) once the MR is merged and the package is released to the GitLab package server
-3. QA:  
+    - Update the version archive in the [upgrade guide](https://github.com/mattermost/docs/blob/master/source/administration/upgrade.md)
```

Comment:
Looks like this is a duplicate from the docs below.

Reply:
Done